platform/ios/src/MGLMapView.h

```diff
@@ -636,11 +637,14 @@ MGL_EXPORT IB_DESIGNABLE
 
  Changing the center coordinate centers the map on the new coordinate without
  changing the current zoom level.
-
+ 
  @param coordinate The new center coordinate for the map.
  @param animated Specify `YES` if you want the map view to scroll to the new
     location or `NO` if you want the map to display the new location
     immediately.
+ 
+ @note The behaviour of this method is undefined if called when observing
```

Comment:
Nit: replace “behaviour” with “behavior”. (Sorry! 🇺🇸)

Reply:
Weird - my Mac is actually set to US, so this must have been me forcing my 🇬🇧!

Reply:
Will change.